src/Elastic.Apm/Helpers/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+namespace Elastic.Apm.Helpers
+{
+	internal static class StringExtensions
+	{
+		internal static string TrimToLength(this string input, int maxLength)
+		{
+			if (input.Length > maxLength)
+				input = $"{input.Substring(0, Consts.PropertyMaxLength-3)}...";
+
+			return input;
+		}
+
+		internal static string TrimToMaxLength(this string input)
```

Comment:
Hi @gregkalapos  i thought about making it private, but i think it can be useful in other parts of code, for that library, for example if we`ll need to trim to custom length or smth like that, that's why i leaved it as internal